ttsfm-web/websocket_handler.py

```
@@ -0,0 +1,231 @@
+"""
+WebSocket handler for real-time TTS streaming.
+
+Because apparently waiting 2 seconds for audio generation is too much for modern users.
+At least this will make it FEEL faster.
+"""
+
+import asyncio
```

Ruff (0.12.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)